pkg/controllers/replica-controller/handler.go

```diff
@@ -257,7 +261,22 @@ func (c *CStorVolumeReplicaController) cVREventHandler(
 		if isCVRCreateStatus(cvrObj) {
 			return c.cVRAddEventHandler(cvrObj, fullVolName)
 		}
-		return c.getCVRStatus(cvrObj)
```

Comment:
Can we move this line after setting the targetIP? Or else it will return the status as `Invalid` for every syncOperation

Reply:
oops. I had it there but did some cleanup before commit... thanks